package.json

```diff
@@ -1,58 +1,12 @@
 {
-  "name": "layout_landing-page",
-  "version": "1.0.0",
-  "description": "Landing page",
+  "name": "landing",
+  "source": "src/index.html",
   "scripts": {
```

Comment:
why you removed the scripts? This is necessary.

The tests are failing and it's missing the DEMO url.